File: services/communal/src/main/java/nc/unc/cs/services/communal/integrations/PaymentRequest.java
package nc.unc.cs.services.communal.integrations;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaymentRequest {

    private Long paymentRequestId;
    private Long serviceId;
    private Long citizenId;
    private Boolean status;
    private Integer amount;
    private Long taxId;
}

File: client-ui/src/models/bank/payment-request.js
export default class PaymentRequest {
    constructor(
        paymentRequestId,
        serviceId,
        citizenId,
        status,
        amount,
        taxId
    ) {
        this.paymentRequestId = paymentRequestId
        this.serviceId = serviceId
        this.citizenId = citizenId
        this.status = status
        this.amount = amount
        this.taxId = taxId
    }
}